### PR TITLE
fix: load flags

### PR DESCRIPTION
**Pull request by mohanson**

### CODE REVIEW

**Comment by mohanson** [comment]
CC @xxuejie @XuJiandong

**Review by xxuejie — APPROVED** [review]

File: c/rce_validator.c [hunk 1]
```diff
@@ -92,27 +92,35 @@ static int make_witness_cursor(uint8_t* buffer, uint32_t cache_size,
 }
 
 int main() {
+  // Load Script args first.
+  //
+  // | Script Args   | Type ID | Flags |
+  // | ------------- |---------| ------|
+  // | Length(Byte)  | 32      | 1     |
+
+  uint8_t current_script[32768];
```
Comment (XuJiandong) [inline c/rce_validator.c:101]: we have such kind macro, instead of magic number.

File: c/rce_validator.c [hunk 2]
```diff
@@ -92,27 +92,35 @@ static int make_witness_cursor(uint8_t* buffer, uint32_t cache_size,
...
+  uint64_t len = 32768;
+  int err = ckb_checked_load_script(current_script, &len, 0);
+  CHECK(err);
+  CHECK2(len <= 32768, ERROR_SCRIPT_TOO_LONG);
```
Comment (XuJiandong) [inline c/rce_validator.c:105]: If the checked version (ckb_checked_load_script) is used, it's not needed to check "len" again here.

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: c/rce_validator.c (hunk 1 above)
Comment (mohanson) [reply c/rce_validator.c:101]: I can't find any macro in ckb-c-std-lib

Comment (XuJiandong) [reply c/rce_validator.c:101]: we can define one "#define SCRIPT_SIZE 32768" :it's frequently used.

Comment (mohanson) [reply c/rce_validator.c:101]: sure

Comment (mohanson) [reply c/rce_validator.c:101]: The script **must** be less than 32768, is this a convention?

File: c/rce_validator.c (hunk 2 above)
Comment (mohanson) [reply c/rce_validator.c:105]: It's wired, you wrote that too 

https://github.com/nervosnetwork/ckb-production-scripts/blob/xudt_rce/c/xudt_rce.c#L231-L233

**Comment by mohanson** [comment]
See https://github.com/nervosnetwork/ckb-production-scripts/pull/13

**Pull request closed without merging**